packages/react-components/react-table/src/components/TableHeaderCell/useTableHeaderCell.tsx

```diff
@@ -6,8 +6,8 @@ import { useTableContext } from '../../contexts/tableContext';
 import { useARIAButtonShorthand } from '@fluentui/react-aria';
 
 const sortIcons = {
-  ascending: <ArrowUpRegular />,
-  descending: <ArrowDownRegular />,
+  ascending: <ArrowUpRegular fontSize={12} />,
+  descending: <ArrowDownRegular fontSize={12} />,
```

Comment:
An icon could be overridden, correct? So...

### Before

```tsx
<TableHeaderCell sortIcon={<FooIcon />} />
```

### After

```tsx
<TableHeaderCell sortIcon={<FooIcon fontSize={12} />} />
```


Should the size be set via styles?

Reply:
If an icon is overridden, then I would expect the user to take full responsibility of sizing, since at that point they are committed to using something that is not our guidance